config/discogs-formats.yml

```diff
@@ -0,0 +1,346 @@
+# carriers

```

Comment:
This file should be in lib/generators/qa/discogs/templates/config only.  There should be instructions on how to run the generator that will put this in an app.

Reply:
I was following the examples of other authorities. These files are all in both locations, though it wasn't clear why:

- authorities.yml
- states.yml
- oclcts-authorities.yml

Reply:
done